storage/client_event_test.go

```diff
@@ -165,7 +165,7 @@ func (ser *storeEventReader) updateCountString() string {
 
 			for _, methodID := range methodIDs {
 				method := proto.Method(methodID)
-				if count, ok := countset[method]; ok {
+				if count, okCount := countset[method]; okCount {
```

Comment:
Is this really necessary? I ask because I see us using the limited scope of such assignments all over the code and it's really a policy decision.